org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/InitializeLaunchConfigurations.java

```diff
@@ -85,6 +81,17 @@ public static String which(String program) {
 		}
 		return res;
 	}
+	
+	private static String getDefaultNodePath() {
+		switch (Platform.getOS()) {
+			case Platform.OS_MACOSX:
+				return "/usr/local/bin/node";
+			case Platform.OS_WIN32:
+				return "C:\\Program Files\\nodejs\\node.exe";
+			default:
+				return "/usr/bin/nodejs";
```

Comment:
@mickaelistria I believe you use a Linux machine, could you please confirm whether this is a sensible default? My Fedora workstation takes forever to start up... 😄

Reply:
@PyvesB On my Fedora machine, it's `/usr/bin/node` but I'm not sure if it's always the case

Reply:
If that's the case, it will probably be similar on Fedora, RHEL and CentOS distributions. I've modified the value accordingly.

Thanks!